runtime/src/snapshot_package.rs

```diff
@@ -21,16 +21,22 @@ pub struct SnapshotPackage {
     pub block_height: Slot,
     pub hash: SnapshotHash,
     pub snapshot_storages: Vec<Arc<AccountStorageEntry>>,
-    pub status_cache_slot_deltas: Vec<BankSlotDelta>,
-    pub bank_fields_to_serialize: BankFieldsToSerialize,
-    pub bank_hash_stats: BankHashStats,
-    pub write_version: u64,
+    pub bank_snapshot_package: BankSnapshotPackage,
 
     /// The instant this snapshot package was sent to the queue.
     /// Used to track how long snapshot packages wait before handling.
     pub enqueued: Instant,
 }
 
+/// A package created from a snapshot request, containing information required to serialize the bank
+/// snapshot
+pub struct BankSnapshotPackage {
+    pub bank_fields: BankFieldsToSerialize,
+    pub bank_hash_stats: BankHashStats,
+    pub status_cache_slot_deltas: Vec<BankSlotDelta>,
+    pub write_version: u64,
+}
```

Comment:
Sorry I wasn't more specific. Can this go after both the struct and impl blocks of SnapshotPackage, please?

Some more discussion here: https://github.com/anza-xyz/agave/pull/9158#discussion_r2546223319